packages/http/src/types.ts

```diff
@@ -206,13 +206,13 @@ export type RouteProducer = (
 export interface HeaderFieldOptions {
   type: "header";
   name: string;
-  format?: "csv";
+  format?: string;
```

Comment:
That should still be typed to the list of options

Reply:
Updated to be a union of strings.